src/pages/home/ReportScreen.js

```diff
@@ -125,6 +128,16 @@ class ReportScreen extends React.Component {
         Report.addAction(getReportID(this.props.route), text);
     }
 
+    /**
+     * Get the currently viewed report ID as number
+     *
+     * @returns {Number}
+     */
+    getReportActionID() {
+        const params = this.props.route.params;
+        return Number.parseInt(params.reportActionID || 0, 10);
```

Comment:
NAB, The `10` second parameter passed to parseInt can be ignored like so `return Number.parseInt(params.reportActionID || 0);`, you can check this out from here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/parseInt

Reply:
so radix param is required as per our eslint. It will throw an error if i remove it so keeping it 🙂 
```
/Users/chirag/Expensidev/App/src/pages/home/ReportScreen.js
error  Missing radix parameter  radix
```